base-lite/ub/ub_test.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func assertEqual(a string, b string, t *testing.T) {
+	if a != b {
+		t.Error(a + " != " + b)
+	}
+}
+
+func TestContains(t *testing.T) {
+	if Contains([]string{"hello", "world"}, "hi") {
+		t.Error(` Contains([]string{"hello", "world"}, "hi") = true`)
+	}
+	if !Contains([]string{"hello", "world"}, "hello") {
+		t.Error(`Contains([]string{"hello", "world"}, "hello") = false`)
+	}
+}
+
+func TestConvertKey(t *testing.T) {
```

Comment:
These are not idiomatic golang tests. Please modify as discussed over zoom call.

Reply:
Added